tools/deploy_s3.py

```diff
@@ -136,8 +144,8 @@ def deploy_to_s3_main(
 
             extra_args = {
                 "CacheControl": cache_control,
-                "ContentEncoding": "gzip",
                 "ContentType": content_type,
+                "ContentEncoding": "identity" if compressed else "gzip",
```

Comment:
Are you sure about this this "identity" option? I can't find any documentation about it.
Maybe just not setting that field would be easier?

Reply:
I think you are right, [RFC](https://www.rfc-editor.org/rfc/rfc2616#section-14.11) says that it should be used only in `Accept-Encoding` header.